backend/controllers/Events.js

```diff
@@ -15,12 +15,25 @@ eventRouter.use(bodyParser.urlencoded({
 eventRouter.get('/list', function(req, res) {
   Event.find(function(err, events) {
     if (err) {
-      res.status(500).json(err);
+      return res.status(500).json(err);
     }
     var mappedEvents = events.map(q => q.toJSONFor());
     res.json(mappedEvents);
   });
-})
+});
+
+eventRouter.get('/event', function(req, res) {
```

Comment:
This would turn out to be `url/event/event', no?  I think it'll be more readable if we just have `url/event/id`.

Reply:
Fixed it